doc/libfuzzer.md

```diff
@@ -0,0 +1,61 @@
+# libfuzzer
+
+## setup 
+
+### Requirements
+libFuzzer needs LLVM sanitizer support, 
+so this only works on x86-64 Linux, 
+x86-64 macOS and Apple-Silicon (aarch64) macOS for now. 
+This also needs a nightly compiler since 
+it uses some unstable command-line flags. 
+You'll also need a C++ compiler with C++11 support.
+
+
+
```

Comment:
Nit: You only need a single blank line here.

Reply:
Done